### PR TITLE
plasma-welcome: Initial inclusion @ 6.1.5

### PR DESCRIPTION
**Pull request by Justinzobel**
**Summary**

Fixes: #3891

**Test Plan**

Opened all pages, used a few launchers.

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/p/plasma-welcome/package.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name       : plasma-welcome
+version    : 6.1.5
+release    : 1
+source     :
+    - https://download.kde.org/stable/plasma/6.1.5/plasma-welcome-6.1.5.tar.xz : 12ece9e7d61c3ec6a23fe9357667416e26802abafdc69f245cc3d96a05ef9d96
+homepage   : https://invent.kde.org/plasma/plasma-welcome
+license    : BSD-3-Clause and GPL-2.0-or-later
```
Comment (davidjharder) [inline packages/p/plasma-welcome/package.yml:7]: We do this as a list:
```
- BSD-3-Clause
- GPL-2.0-or-later
```

The checks don't understand the lists, but that's fine

**Review by davidjharder — CHANGES_REQUESTED** [review]
You have some Actions warnings

File: packages/p/plasma-welcome/package.yml (hunk 1 above)
Comment (Justinzobel) [reply packages/p/plasma-welcome/package.yml:7]: I took it directly from Fedora and they use a single line haha oops. Honestly the licensing stuff is confusing and even Nate who is the main developer of Plasma Welcome is partially confused by it.

If anyone is skilled at licensing stuff please verify I have got it right.

**Comment by davidjharder** [comment]
- Lol, now you've made the checks angrier. `release` should be 1
- Your fix-up commit needs to be squashed.

**Comment by Justinzobel** [comment]
Machines hate me :D

**Review by davidjharder — APPROVED** [review]

**Pull request merged**